apps/web-giddh/src/app/settings/profile/setting.profile.component.ts

```diff
@@ -276,6 +283,9 @@ export class SettingProfileComponent implements OnInit, OnDestroy {
 
         this.route.params.pipe(takeUntil(this.destroyed$)).subscribe(params => {
             this.currentTab = (params['referrer']) ? params['referrer'] : "personal";
+            if ((params['referrer']) === 'personal') { this.activeTabIndex = 0; }
+            else if ((params['referrer']) === 'address') { this.activeTabIndex = 1; }
```

Comment:
format code

Reply:
already formatted, but still format in clear way